x/multisig/context_test.go

```diff
@@ -48,6 +48,13 @@ func TestContext(t *testing.T) {
 			[]weave.Address{other},
 			[]weave.Address{sig, random},
 		},
+		{
+			// add multisig conditions for both contractID and otherContractID to the context
+			func() weave.Context { return withMultisig(withMultisig(bg, contractID), otherContractID) }(),
```

Comment:
why wrap in this func, not just `withMultisig(withMultisig(bg, contractID), otherContractID)` like you do below